src/daemon/config.rs

```diff
@@ -215,6 +217,12 @@ impl Config {
         let config_file = matches
             .value_of("config")
             .unwrap_or("./defaults/krill.conf");
+        format!("{}", config_file)
+    }
+
+    /// Creates the config (at startup). Panics in case of issues.
+    pub fn create() -> Result<Self, ConfigError> {
+        let config_file = &*Self::get_config_filename();
```

Comment:
Why do we need &* here?

Reply:
actually I'm a newbie in rust world... I will fix this